packages/app_ui/pubspec.yaml

```diff
@@ -10,6 +10,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  google_fonts: ^6.2.1
```

Comment:
for what is that?

Reply:
For my font style (poppins) i use it from google_fonts package